dt/uni07eta/networker/kustomization.yaml

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
IMHO, we should split this into `nodeset` and `deployment`.

Reply:
splitted